docs/en/ingest-management/elastic-agent/run-container-common/download-elastic-agent.asciidoc

```diff
@@ -34,6 +34,6 @@ endif::[]
 For Kubernetes Security Posture Management (KSPM) purposes, the {agent} requires read access to various types of Kubernetes resources, node processes, and files.
 To achieve this, read permissions are granted to the {agent} for the necessary resources, and volumes from the hosting node's file system are mounted to allow accessibility to the {agent} pods.
 
-TIP: The size and the number of nodes in a Kubernetes cluster can be fairly large at times, and in such cases the Pod that will be collecting cluster level metrics might face performance issues due to resources limitations. In this case users might consider to avoid using the leader election strategy and instead run a dedicated, standalone {agent} instance using a Deployment in addition to the DaemonSet.
+TIP: The size and the number of nodes in a Kubernetes cluster can be fairly large at times, and in such cases the Pod that will be collecting cluster level metrics might face performance issues due to resources limitations. In this case users might consider to avoid using the leader election strategy and instead run a dedicated, standalone {agent} instance using a Deployment in addition to the DaemonSet. For more information check <<scaling-on-kubernetes>> page.
```

Comment:
TIP: The size and the number of nodes in a Kubernetes cluster can be fairly large at times, and in such cases the Pod that will be collecting cluster level metrics might require more runtime resources than you would like to dedicate to all of the pods in the DaemonSet. If under resourced the leader which is collecting the cluster wide metrics may face performance issues due to resource limitations. In this case users might consider to avoid using a single DaemonSet and the leader election strategy and instead run a dedicated, standalone {agent} instance for collecting cluster wide metrics using a Deployment in addition to the DaemonSet to collect metrics for each node. Then both the Deployment and the DaemonSet can be resourced appropriately. For more information check the <<scaling-on-kubernetes>> page.